src/null-safety/understanding-null-safety/index.md

```diff
@@ -726,9 +726,9 @@ redundantly check it again for `null`:
 
 ```dart
 // Using null safety:
-checkList(List? list) {
+String checkList(List? list) {
   if (list == null) return 'No list';
-  if (list?.isEmpty) {
+  if (list?.isEmpty ?? false) {
```

Comment:
While you're at it, line 705 needs the same fix.

Also, would you mind changing `List` to `List<Object>`? I think we generally try to avoid raw types in docs.

Reply:
Good suggestions, thanks. I've added the generic type and fixed the other example as well.